extensions/emmet/src/test/wrapWithAbbreviation.test.ts

```diff
@@ -63,6 +63,22 @@ const wrapInlineElementExpectedFormatFalse = `
 	</ul>
 `;
 
+const wrapMultiLineJsxExpected = `
+	<ul class="nav main">
+		<div className="hello">
+			<li class="item1">img</li>
+			<li class="item2">$hithere</li>
+		</div>
+	</ul>
+`;
+
+const wrapIndividualLinesJsxExpected = `
+	<ul class="nav main">
+		<div className="hello1"><li class="item1">img</li></div>
+		<div className="hello2"><li class="item2">$hithere</li></div>
+	</ul>
+`;
+
```

Comment:
Can you put these together with their tests?

Reply:
Absolutely!